### PR TITLE
Axrandomgenerator implement

### PR DESCRIPTION
**Pull request by LinqingZ**

### CODE REVIEW

File: aepsych/generators/random_generator.py [hunk 1]
```diff
@@ -59,3 +60,42 @@ def from_config(cls, config: Config):
...
+        self.bounds_ = torch.stack([self.lb, self.ub])
+
+    @classmethod
+    def get_config_options(cls, config: Config, name: str) -> Dict:
+        
+        # config_opts = super().get_config_options(config, name)
+        # lb = 
+        opts = {
+            "model": cls.model,
+        }
```
Comment (crasanders) [inline aepsych/generators/random_generator.py:90]: You should also have key named `"model_kwargs"` in this dict. The value should be the arguments that `Models.UNIFORM` takes. I think at a minimum it takes a seed, but you should double check the Ax documentation. Look at what `AxSobolGenerator` does.

File: tests/generators/test_random_generator.py [hunk 2]
```diff
@@ -59,3 +59,25 @@ def test_randomgen_config(self):
...
+                outcome_types = [continuous]
+                strategy_names = [init]
+
+                [init]
+                generator = RandomGenerator
+                """
+        config = Config(config_str=config_str)
+        gen = AxRandomGenerator.from_config(config)
+        self.assertEqual(gen.model, Models.UNIFORM)
+        self.assertEqual(gen.dim, len(gen.lb))
```
Comment (crasanders) [inline tests/generators/test_random_generator.py:78]: You don't need this assert since the generator doesn't need to know the bounds or dimensionality under the Ax API. You should make sure it correctly reads the seed or any other arguments `Models.UNIFORM` might take, though.

**Review by crasanders — CHANGES_REQUESTED** [review]

**Review by crasanders — CHANGES_REQUESTED** [review]
Please remove the changes to the notebooks from this PR. Otherwise this looks good.

File: tests/generators/test_random_generator.py [hunk 3]
```diff
@@ -59,3 +59,29 @@ def test_randomgen_config(self):
...
+                [RandomGenerator]
+                seed=231
+                deduplicate=True
+                """
+        config = Config(config_str=config_str)
+        gen = AxRandomGenerator.from_config(config, name="init")
+        self.assertEqual(gen.model, Models.UNIFORM)
+        self.assertEqual(gen.model_kwargs["seed"], 231)
+        self.assertTrue(gen.model_kwargs["deduplicate"])
+
```
Comment (mshvartsman) [inline tests/generators/test_random_generator.py:83]: Here I would also test that the generator obeys parameter bounds (this can often catch shape and scale errors).

**Review by mshvartsman — CHANGES_REQUESTED** [review]
Please address this comment: https://github.com/facebookresearch/aepsych/pull/243/files#r1146888920. The test should verify that the generator obeys bounds (this implicitly also tests that the dimensions of the output arrays are correct and nothing got tranposed/scrambled, which has happened before). 

Otherwise looks okay to me. @crasanders is there any reason we might want to combine the model-less generators (random and sobol) into a single class? Seems like the main difference is what we pick from the ax `Models` enum.
